<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreationWindow</class>
 <widget class="QDialog" name="CreationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Criar</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>19</y>
     <width>221</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string>Nova turma</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Código da Disciplina</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="discipline">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="classId">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Turma</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createClass">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Criar</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>221</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string>Novo aluno</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nome</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="studentName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="studentId">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Matrícula</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="disciplineStudent">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Disciplina</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Turma</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="classIdStudent">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="createStudent">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>310</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Criar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="readTag">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Exibir Tag</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tag</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tagId">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <zorder>label_4</zorder>
   <zorder>studentName</zorder>
   <zorder>studentId</zorder>
   <zorder>label_6</zorder>
   <zorder>classId_2</zorder>
   <zorder>label_8</zorder>
   <zorder>discipline_2</zorder>
   <zorder>label_7</zorder>
   <zorder>disciplineStudent</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>classIdStudent</zorder>
   <zorder>createStudent</zorder>
   <zorder>readTag</zorder>
   <zorder>label_5</zorder>
   <zorder>tagId</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
